.travis.yml

```diff
@@ -64,11 +73,14 @@ before_script:
     fi
   # the content tests are intensive and there are memory leaks, this is more pronounced with the Jackalope DBAL PHPCR implementation.
   - echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
-  - phpenv config-rm xdebug.ini
+  - phpenv config-rm xdebug.ini || true
   - composer self-update
+  - if [[ $COMPOSER_DEV_STABILITY = true ]]; then composer config minimum-stability dev ; fi
   - if [[ $PHPCR_TRANSPORT = jackrabbit ]]; then composer require jackalope/jackalope-jackrabbit:~1.2 --no-update --no-interaction ; fi
   - composer validate --strict
   - composer update -o $COMPOSER_FLAGS
+  # Following hack on dev stability can be removed when https://github.com/FriendsOfSymfony/FOSRestBundle/pull/2023 is merged
+  - if [[ $COMPOSER_DEV_STABILITY = true ]]; then php -r "file_put_contents('vendor/friendsofsymfony/rest-bundle/Controller/ExceptionController.php', str_replace('FlattenException::create(', 'FlattenException::createFromThrowable(', file_get_contents('vendor/friendsofsymfony/rest-bundle/Controller/ExceptionController.php')));" ; fi
```

Comment:
This is a hack to implement https://github.com/FriendsOfSymfony/FOSRestBundle/pull/2023 else most of the Rest 404 tests fail. Currently just to check what else is failing in the CI.